src/Fls.Results/Fls.Results.csproj

```diff
@@ -2,6 +2,21 @@
 
   <PropertyGroup>
     <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
We might want to lower the netstandard version. We can do better than 2.0. 1.0 would be ideal but we can settle for the lowest version possible.